src/expandableCalendar/week.tsx

```diff
@@ -56,6 +56,11 @@ const Week = React.memo((props: WeekProps) => {
   const dayProps = extractDayProps(props);
   const currXdate = useMemo(() => parseDate(current), [current]);
 
+  const getDayState = (day: XDate) => {
+    const isControlled = disableDaySelection;
```

Comment:
let's move it back to state-manager to encapsulate this logic

Reply:
Done